tests/testthat/test-rd-markdown.R

```diff
@@ -258,8 +254,6 @@ test_that("markdown is parsed in all fields where it is supported", {
     #'
     #' @seealso See also **with bold**
     #'
-    #' @keywords Keywords **with bold**
-    #'
```

Comment:
```r
roc_proc_text(rd_roclet(), "#' Title.\n#'\n#' @keywords Keywords **with bold**\nfoo <- function() {}")
```
$foo.Rd
% Generated by roxygen2: do not edit by hand
% Please edit documentation in Rtmpky86Q9/file53cac0b532b
\name{foo}
\alias{foo}
\title{Title.}
\usage{
foo()
}
\description{
Title.
}
\keyword{**with}
\keyword{Keywords}
\keyword{bold**}
```